types/apps/index.d.ts

```diff
@@ -48,7 +48,7 @@ export type AppRouteData = AppRoute & {
 
 export type BadgeInfo = {
 	show: boolean;
-	count?: number;
+	count: number | string;
```

Comment:
I don't feel good about making this field required (which implies a breaking change btw).

Also, if we allow the string type, the commit should be marked as feat, since we are adding a new feature